2023-09-13 22:43:35: 0, failed to connect to DBServer: 127.0.0.1:23001
2023-09-13 22:44:19: 0, failed to connect to DBServer: 127.0.0.1:23001
2023-09-13 23:09:54: 
AuxiliaryTimer took much times to process: 37496ms
2023-09-13 23:09:59: 
AuxiliaryTimer took much times to process: 2578ms
2023-09-13 23:10:00: 
AuxiliaryTimer took much times to process: 1125ms
2023-09-13 23:10:02: 
AuxiliaryTimer took much times to process: 1415ms
2023-09-13 23:10:03: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:11:04: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:12:04: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:12:12: 
AuxiliaryTimer took much times to process: 1555ms
2023-09-13 23:13:06: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:14:06: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:15:06: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:16:06: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:17:06: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:18:07: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:19:07: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:20:07: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:21:07: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:22:07: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:23:07: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:24:07: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:25:08: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:26:08: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:27:08: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:28:08: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:29:08: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:30:08: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:31:08: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:32:09: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:33:09: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:34:09: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:35:09: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:36:09: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:37:09: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:38:10: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:39:10: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:40:10: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:41:10: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:42:10: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:43:10: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:44:11: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:45:11: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:46:11: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:47:11: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:48:11: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:49:11: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:50:11: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:51:12: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:52:12: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:53:12: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:54:12: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:55:12: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:56:12: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:57:13: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:58:13: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
2023-09-13 23:59:13: Update server Status Bug :System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 207.148.73.199:88
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.UploadDataInternal(Uri address, String method, Byte[] data, WebRequest& request)
   at System.Net.WebClient.UploadData(Uri address, String method, Byte[] data)
   at GameServer.VLTK.Core.Activity.X2ExpEvent.ExpMutipleEvent.UpdateServerStatus() in E:\OpenAI\Resource_SELL_2\Resource_SELL\Server\GameServer\KTGameServer\GameServer\VLTK\Activity\X2ExpEvent\ExpMutipleEvent.cs:line 160
